_includes/sass/custom/_commandpalette.scss

```
@@ -0,0 +1,34 @@
+.dbt-cloud-command-palette__search-result-row {
+    padding: 10px;
+    border-top: 1px solid #e6e8ed;
```

These colors could likely be theme variables (`_variables.scss`)
```suggestion
    border-top: 1px solid $gray;
```